content/en/docs/instrumentation/net/manual.md

```diff
@@ -77,13 +78,20 @@ for ASP.NET Core setup.
 First, ensure that you have the right packages:
 
 ```
-dotnet add package OpenTelemetry --prerelease
-dotnet add package OpenTelemetry.Extensions.Hosting --prerelease
-dotnet add package OpenTelemetry.Exporter.Console --prerelease
+dotnet add package OpenTelemetry
+dotnet add package OpenTelemetry.Extensions.Hosting
+dotnet add package OpenTelemetry.Exporter.Console
+```
+
+Then you can install the Instrumentation package
+
+```
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
 ```
 
 Note that the `--prerelease` flag is required for all instrumentation packages
```

Comment:
This is not true for Instrumentation.Runtime. Not sure if that is worthy.

Reply:
True, I think I'd stand by the statement as caveats would be confusing here. Saying that only part of it isn't prerelease would only affect the minority, and would be a pleasant surprise for them.